### PR TITLE
Update linting

### PR DESCRIPTION
**Pull request by gparvin**

### CODE REVIEW

File: build/common/config/.golangci.yml [hunk 1]
```diff
@@ -68,7 +78,8 @@ linters-settings:
     # report about shadowed variables
     check-shadowing: false
   gci:
-    local-prefixes: open-cluster-management.io/governance-policy-template-sync
+    sections:
+      - prefix(open-cluster-management.io/config-policy-controller)
```
Comment (dhaiducek) [inline build/common/config/.golangci.yml]: This path should match the repo 🙂

**Review by dhaiducek — CHANGES_REQUESTED** [review]
Thanks, @gparvin! Would you please rebase this PR? There's also one minor update needed.

File: build/common/config/.golangci.yml (hunk 1 above)
Comment (gparvin) [reply build/common/config/.golangci.yml]: Done, thanks

**Comment by gparvin** [comment]
@dhaiducek The go commit is something that was missed I think from the previous go PR.

**Review by mprahl — APPROVED** [review]

**Review by dhaiducek — APPROVED** [review]

**Pull request merged**